index.js

```diff
@@ -991,7 +992,25 @@ module.exports = function (log, indexesPath) {
     } else if (!op.type) {
       // to support `query(fromDB(jitdb), toCallback(cb))`
       getFullBitset(cb)
-    } else console.error('Unknown type', op)
+    } else if (op.type === 'DEFERRED') {
+      // DEFERRED only appears in this pipeline when using `prepare()` API
+      op.task(
+        { jitdb },
+        (err, newOp) => {
+          if (err) {
+            console.error('Error executing DEFERRED operation', err)
```

Comment:
why not bubble up the error here?

Reply:
The `cb` is not `cb(err, result)`, it's `cb(bitset, filters)`, unfortunately

Reply:
right, then this is fine. But would probably be good to change :)